commit/factory.go

```diff
@@ -230,6 +231,7 @@ func (p *PluginFactory) NewReportingPlugin(ctx context.Context, config ocr3types
 		offchainConfig.TokenInfo,
 		ccipReader,
 		offchainConfig.PriceFeedChainSelector,
+		cache.NewCache[string, uint8](cache.NeverExpirePolicy{}),
```

Comment:
Are you imagining the cache would be configured by the plugin configuration? I'm not sure why this is surfaced to the factory rather than left as an implementation detail.

Reply:
I think we can leave this as an implementation detail. I moved the the cache instantiation in the price reader constructor.